src/docfx/template/TemplateEngine.cs

```diff
@@ -204,6 +204,7 @@ private RenderType GetTocRenderType()
             }
 
             var jsonSchema = JsonUtility.DeserializeData<JsonSchema>(schemaString, new FilePath(schemaFilePath));
+            jsonSchema.SchemaType = mime.ToLowerInvariant();
```

Comment:
What is the purpose of to lower here? We typically preserve the case of mime.

Reply:
OK.